llarp/service/endpoint.cpp

```diff
@@ -457,36 +457,42 @@ namespace llarp
         m_OnReady->Stop();
     }
 
-    bool
-    Endpoint::PublishIntroSet(const EncryptedIntroSet& i, AbstractRouter* r)
+    template < typename Endpoint_t >
+    static absl::optional< path::Path::UniqueEndpointSet_t >
+    GetManyPathsWithUniqueEndpoints(Endpoint_t* ep, size_t N, size_t tries = 10)
     {
```

Comment:
The `UniqueEndpoints` part of this method doesn't seem to be enforced at all, seems like this is more of `GetPathsToEndpoint` -- or am I missing something?

Reply:
the `path::Path::UniqueEndpointSet_t` takes care of that part.